DataFormats/Math/interface/GeantUnits.h

```diff
@@ -32,6 +32,8 @@ namespace geant_units {
     constexpr long double operator"" _m(long double length) { return length * 1000.; }
     constexpr long double operator"" _cm3(long double length) { return length * 1._cm * 1._cm * 1._cm; }
     constexpr long double operator"" _m3(long double length) { return length * 1._m * 1._m * 1._m; }
+    constexpr long double operator"" _mm(unsigned long long int length) { return length * 1; }
+    constexpr long double operator"" _cm(unsigned long long int length) { return length * 10; }
 
```

Comment:
Shouldn't integer arithmetic be preserved in these cases?
```
constexpr unsigned long long int operator"" _mm(unsigned long long int length) { return length * 1; }
```

Reply:
This is consistent with what we already have:
https://cmssdt.cern.ch/lxr/source/DataFormats/Math/interface/deltaPhi.h?v=CMSSW_11_0_X_2019-07-01-2300#0065